src/administration/helpers.ts

```diff
@@ -45,3 +45,6 @@ export const redactProjectKeyValue = (key: string) =>
     key.substring(API_KEY_PREFIX.length + 2, key.length - 2),
     '*'.repeat(key.length - 12)
   );
+
+export const redactKey = (key: string) =>
+  key.replace(key.substring(2, key.length - 2), '*'.repeat(10));
```

Comment:
Suggestion:

I'd add length guard that requires some fixed minimum size of the key to ensure large enough segment is redacted. In case of the key being too short, we can either fail or redact the whole thing.